src/app/catalog/components/issuer-catalog/issuer-catalog.component.html

```diff
@@ -1,319 +1,105 @@
-					</label>
-				</div>
-			</div>
-
-			<ng-container *ngIf="badgesDisplay == 'grid'">
-				<ng-container *ngIf="!groupByCategory">
-					<div class="issuercard" *ngFor="let issuer of issuerResults">
-						<div class="issuercard-x-image">
-							<img
-								[loaded-src]="issuer?.image"
-								[loading-src]="issuerPlaceholderSrc"
-								[error-src]="issuerPlaceholderSrc"
-								alt="{{ issuer.name }} avatar"
-							/>
-						</div>
-						<div class="issuercard-x-title">
-							<a (click)="navigateToIssuer(issuer)">{{ issuer.name }}</a>
-							<!-- <p><strong>Your Role: </strong>{{ issuer.currentUserStaffMember?.roleInfo.label }}</p> -->
-						</div>
-						<div class="issuercard-x-desc" [truncatedText]="issuer.description" [maxLength]="250"></div>
-
-						<!-- <div class="issuercard-x-topbadges" *ngFor="let issuerBadges of [issuerToBadgeInfo[issuer.issuerUrl]]">
-							<h3 class="u-text-body-bold u-text-dark1 u-margin-bottom2x" *ngIf="issuerBadges">Top Badges:</h3>
-							<div class="l-flex l-flex-2x l-flex-column"></div>
-						</div> -->
-
-						<div class="issuercard-x-footer">
-							<div class="l-flex l-flex-aligncenter l-flex-1x">
-								<svg
-									class="icon icon-light4 u-hidden-maxmobilelarge"
-									width="24px"
-									icon="icon_badge"
-								></svg>
-								<p
-									*ngIf="issuer.badgeClassCount"
-									class="u-text-small u-text-nowrap"
-									[innerHTML]="issuer.badgeClassCount | i18nPlural: plural['badges']"
-								></p>
-							</div>
-							<a class="u-text u-text-small" [routerLink]="['/public/issuers/', issuer.slug]">{{
-								'Issuer.open' | translate
-							}}</a>
-						</div>
-					</div>
-				</ng-container>
-
-				<ng-container *ngIf="groupByCategory">
-					<ng-container *ngFor="let issuerGroup of issuerResultsByCategory">
-						<h3 class="u-text-h3-semibold u-margin-bottom2x u-margin-top6x u-text-dark1">
-							{{ issuerKeys[issuerGroup.category] }}
-							<span class="u-text-small-semibold-caps u-text-dark4 u-margin-left2x"
-								>{{ issuerGroup.issuers.length }}
-								{{ issuerGroup.issuers.length == 1 ? 'Institution' : 'Institutionen' }}</span
-							>
-						</h3>
-
-						<div class="issuercard" *ngFor="let issuer of issuerGroup.issuers">
-							<div class="issuercard-x-image">
-								<img
-									[loaded-src]="issuer?.image"
-									[loading-src]="issuerPlaceholderSrc"
-									[error-src]="issuerPlaceholderSrc"
-									alt="{{ issuer.name }} avatar"
+						<div class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-gap-6">
+							<div class="tw-flex tw-relative tw-items-center">
+								<input
+									hlmInput
+									class="tw-bg-white tw-w-full tw-border-1 tw-border-[#492E98] md:tw-w-96 tw-border-solid tw-min-h-12 tw-rounded-lg tw-p-2 tw-font-[rubik]"
+									placeholder="{{ 'Issuer.searchInstitutions' | translate }}"
+									[(ngModel)]="searchQuery"
 								/>
+								<hlm-icon
```

Comment:
maybe we should add icon option to oeb-input

Reply:
would be nice, i have not found any other input where an icon is needed but good to keep in mind for the future